study.md

```diff
@@ -24,7 +24,13 @@ material. Instead, digest what you've read and repeat it in your own voice.
 In your own words, how Ember "stops breaking the web".
 
 ```md
-<!-- your answer here -->
+Ember loves URLS and so should you. You've been breaking the internet because
+you're a hot shot JS dev who uses client-side frameworks and hashbang
+URL's to load current-state and navigates easily cuz jQuery bro. But your hashbangs
+and terrible URL's are slowing down your users experience (especially those with older
+browsers, who don't even want your fancy new feature's that won't load) and making it
+hard for people to know where to go to get to your website. Good luck having some
+one bookmark your page.
```

Comment:
Yep, URLs are awesome. They let you share your page, bookmark it, or even go back using the back button. Single-page apps are great too, but they sacrifice URLs. Ember gets the best of both worlds.